x/upgrade/client/rest/query.go

```diff
@@ -0,0 +1,41 @@
+package rest
+
+import (
+	"net/http"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/types/rest"
+	upgrade "github.com/cosmos/cosmos-sdk/x/upgrade/internal/types"
+	"github.com/gorilla/mux"
+)
+
+// RegisterRoutes registers REST routes for the upgrade module under the path specified by routeName.
+func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router, cdc *codec.Codec, storeName string) {
+	r.HandleFunc("/"+upgrade.ModuleName, getUpgradePlanHandler(cdc, cliCtx, storeName)).Methods("GET")
```

Comment:
I noticed that the REST txs commands are not defined. Any reason why? I believe we should add them too.

Reply:
I don't really know how this works for the client, and so far it was just cli tooling for us but sure I can add them.